packages/utils/src/simple-logger-example.ts

```diff
@@ -0,0 +1,24 @@
+import { getLogger, log, setLogLevel } from "./logger.ts";
```

Comment:
I think jsdocs and tests cover everything presented here -- disconnected source files have an opportunity for becoming out of date